src/pudl/metadata/fields.py

```diff
@@ -1970,6 +1982,15 @@
         "description": "The energy contained in fuel burned, measured in million BTU.",
         "unit": "MMBtu",
     },
+    "hour_of_year": {
+        "type": "number",
+        "description": "Number between 1 and 8670 representing the hour in a given year.",
+    },
+    "hour_utc": {
+        "type": "datetime",
+        "description": "Hourly datetime in UTC.",
+        "unit": "UTC",
+    },
```

Comment:
There's a similar field for CEMS, but it's called `operating_hour_utc` which felt a little different.

Reply:
IIRC the EPA CEMS "operating hour" was an integer hour-of-day, and it's only present in the raw input and not the outputs. We do have an `operating_datetime_utc` in CEMS, but that's a relic we should really rename to be consistent with all of the other hourly data where we're using `datetime_utc`